client/landing/gutenboarding/onboarding-block/design-selector/style.scss

```diff
@@ -1,4 +1,22 @@
 .design-selector {
+	h1 {
+		color: #23282d;
+		font-size: 2em;
+		font-weight: bold;
+		margin-bottom: 0.25em;
+		text-align: center;
+	}
+
+	h2,
+	h3 {
+		color: #23282d;
```

Comment:
Are there any sass variables available for these?

Reply:
Yep: 0b4134de7d